pytube/helpers.py

```diff
@@ -88,7 +88,7 @@ def safe_filename(s: str, max_length: int = 255) -> str:
     return filename[:max_length].rsplit(" ", 0)[0]
 
 
-def setup_logger(level: int = logging.ERROR):
+def setup_logger(level: int = logging.ERROR, log_filename: str = "pytube.log"):
```

Comment:
I don't think we should default to logging to a file, especially since this is a breaking change. Maybe make the log_filename optional, and if it exists, have setup_logger log to a file instead of stdout.

Reply:
I was also considering that - I think, making it optional is the least breaking change ...